### DISCUSSION TITLE
Is there a way to signin without creating the session file?

### DISCUSSION THREAD

**Discussion by doffn** [mahrtayyab/tweety#177] [Q&A]
I am tring to use tweety to get tweets using my cookies in my django app. I cannot create the session file and accessing the session file as static file is complicated. So can i login to my account without using the session file.

**Comment by mahrtayyab** [comment]
`load_cookies` unlike `sign_in` or `start` don't checks for session file, you can always do `load_cookies` and it will load those cookies.

**Reply by doffn** [reply]
Thank you for the response. You are right, but before calling the cookies in this  `app = Twitter("session")` the session file must be present. It will give a session error in the absence of the file.

**Reply by mahrtayyab** [reply]
It will create the file if doesn't exists

**Comment by mahrtayyab** [comment]
It is now , use `MemorySession()` instead of session name

**Reply by doffn** [reply]
Can i directly use the cookie_value without needing to make a session file. My calls seems to not work can you send me how to use `MemorySession()`.

**Reply by mahrtayyab** [reply]
https://github.com/mahrtayyab/tweety/wiki/FAQs#memory-session
